### PR TITLE
{bio}[foss/2022a] cuteSV v2.0.3 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
Test report by @PetrKralCZ
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4005.donphan.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 530.30.02, Python 3.6.8
See https://gist.github.com/PetrKralCZ/3cf34617174e9ac0870de60d69aff794 for a full test report.

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/f8bab195a0d8e2f874ab68eadbf24ca6 for a full test report.

File: easybuild/easyconfigs/c/cuteSV/cuteSV-2.0.3-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('cigar', '0.1.3', {
+        'checksums': ['5847f5e8968035b3a5b04dcfa879fb6c14dd3a42dce8994864806dcda8a4fcf2'],
+    }),
+    (name, version, {
+        'modulename': '%(name)s',
```
[the remark this answers is not in the document]
Comment (PetrKralCZ) [reply easybuild/easyconfigs/c/cuteSV/cuteSV-2.0.3-foss-2022a.eb:28]: If I don't put it there, it calls `import cutesv` but `import cuteSV` needs to be called there.

[the remark this answers is not in the document]
Comment (verdurin) [reply easybuild/easyconfigs/c/cuteSV/cuteSV-2.0.3-foss-2022a.eb:28]: Understood, thanks.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**